modules/ai/connect/ai.php

```diff
@@ -84,6 +84,32 @@ public function get_remote_frontend_config( $data ) {
 		);
 	}
 
+	/**
+	 * @param array $event_data {
+	 *     @type string $name
+	 *     @type array $data
+	 *     @type array $client {
+	 *         @type string $name
+	 *         @type string $version
+	 *         @type string $session_id
+	 *     }
+	 * }
+	 */
+	public function send_event( array $event_data ) : void {
+		$this->ai_request(
```

Comment:
Can this block the editor in case the service goes down?

Reply:
No, it happens after the import kit settings Ajax call, so the editor will work as usual.